### DISCUSSION TITLE
Welcome to ussd Discussions!

### DISCUSSION THREAD

**Discussion by saulchelewani** [saulchelewani/truroute-ussd-adapter#12] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Fraganya** [comment]
Hi, I am  Francis.

I am the Technology Lead at [Lync Systems](www.lyncsystems.tech) , a Malawian Tech startup. We are the using ussd lib in one of our products.
